pkg/services/validator/aws.go

```diff
@@ -288,6 +676,10 @@ func readAwsCredentials(c *components.AWSPluginConfig, k8sClient kubernetes.Inte
 func initAwsRule[R awsRule](r R, ruleType string, ruleNames *[]string) error {
 	name := reflect.ValueOf(r).Elem().FieldByName("Name").String()
 	if name != "" {
+		// not all awsRules have a Name field, for now we can create a unique name for them
+		if name == "<invalid Value>" {
```

Comment:
Where is `"<invalid Value>"` coming from?

Reply:
This is coming from the reflect package. Unlike tag rules and service quota rules, all the IAM based rules dont have a `Name` field, so the relfect package ends up returning the string `<invalid Value>`.

One option would be to give the IAM rules a `Name` field, but i wanted to avoid making another tiny change that would involve updating the plugin, the validator, the palette-cli, and the validatorctl. Instead i opted for us generating a rule name.

Let me know what you think.